scripts/addons/io_scene_gltf2/gltf2_generate.py

```diff
@@ -2561,87 +2564,29 @@ def generate_materials(operator,
                     else:
                         alphaMode = 'BLEND'
 
-                common['diffuseFactor'] = [blender_material.diffuse_color[0] * blender_material.diffuse_intensity,
+                pbrMetallicRoughness['baseColorFactor'] = [blender_material.diffuse_color[0] * blender_material.diffuse_intensity,
```

Comment:
what about adding default fallback pbt parameters? `metallicFactor: 0, roughnessFactor: 0.5`

Reply:
Good point; I think `roughnessFactor >= 0.5` is the key thing, so subjectively something like ~0.9 would be about right..

Reply:
:+1:

Reply:
In any case it should be consistent with the material_unlit extension where the recommended value for roughness is 0.5, so you could probabliy specify there >=0.5 or set directly 0.9

Reply:
I don't think we're recommending 0.5 anywhere in the extension... it says just:

> * `roughnessFactor` is greater than `0.5`.

Reply:
Ah you're right I just misread that part :)